apps/src/code-studio/pd/teacher_application/SummerProgramContent.jsx

```diff
@@ -20,30 +18,23 @@ export default React.createClass({
       fallbackSummerWorkshops: PropTypes.string
     }).isRequired,
     selectedWorkshop: PropTypes.string
-  },
+  };
+
+  constructor(props) {
+    super(props);
 
-  getInitialState() {
-    return {
+    this.state = {
       dialogWasDismissed: false,
     };
-  },
-
-  checkListChange(event) {
-    const selectedButtons = $(`[name=${event.target.name}]:checked`).map( (index, element) => element.value).toArray();
-    this.props.onChange({[event.target.name]: selectedButtons});
-  },
-
-  radioButtonListChange(event) {
-    this.props.onChange({[event.target.name]: event.target.value});
-  },
```

Comment:
These apparently weren't used. Also this entire `teacher_application` directory is for last year's application and is deprecated. We have a tech debt item to delete it.

Reply:
How much additional work is it to just delete these components now, instead of upgrading them to ES6 classes?

Reply:
Good question. Probably not a whole lot. I can take a look later today, after my regularly scheduled tasks and report back :)